annotate.el

```diff
@@ -403,41 +403,41 @@ See `annotate-blacklist-major-mode'."
   "Calculate an hash for the argument `OBJECT'."
   (secure-hash 'md5 object))
 
-(defun annotate-end-of-line-pos ()
+(define-inline annotate-end-of-line-pos ()
  "Get the position of the end of line and rewind the point's
 position (so that it is unchanged after this function is called)."
-  (save-excursion
-    (end-of-line)
-    (point)))
+ (inline-quote (line-end-position)))
 
-(defun annotate-beginning-of-line-pos ()
+(define-inline annotate-beginning-of-line-pos ()
   "Get the position of the beginning of line and rewind the point's
 position (so that it is unchanged after this function is called)."
-  (save-excursion
-    (beginning-of-line)
-    (point)))
+  (inline-quote (line-beginning-position)))
```

Comment:
Cool, I didn't even know this function existed!

Reply:
me too! :smile: 

Also this is a suggestion coming from a kind person on internet. :)